core/src/main/scala/com/databricks/labs/remorph/utils/ParsingUtils.scala

```diff
@@ -0,0 +1,15 @@
+package com.databricks.labs.remorph.utils
+
+import org.antlr.v4.runtime.misc.Interval
+
+object ParsingUtils {
+
+  /**
+   * Creates a string representation of the text represented by the given ANTLR ParserRuleContext.
+   *
+   * Note that this should exactly reflect the original input text as bounded by the source interval
+   * recorded by the pareser.
+   */
+  def getTextFromParserRuleContext(ctx: org.antlr.v4.runtime.ParserRuleContext): String =
```

Comment:
```suggestion
  def getTextFromParserRuleContext(ctx: ParserRuleContext): String =
```
it might be better like this

Reply:
Ther was some reason I spelled it out, but I no longer remember - can do though.